_data/ome2021.json

```
@@ -273,10 +355,56 @@
         "@type": "VirtualLocation",
         "url": "TBD"
       },
+      "performer": [
+        {
+          "@type": "Person",
+          "name": "Stefanie Weidtkamp-Peters",
+          "affiliation": {
+            "@type": "Organization",
+            "name": "Heinrich-Heine University Duesseldorf"
+          },
+          "sameAs": "https://www.cai.hhu.de/en/team"
+        },
+        {
+          "@type": "Person",
+          "name": "Evan Sander",
+          "affiliation": {
+            "@type": "Organization",
+            "name": "Harvard Medical School"
+          },
+          "sameAs": "https://meded.hms.harvard.edu/people/evan-sanders"
+        },
+        {
+          "@type": "Person",
+          "name": "Luca Lianas",
+          "affiliation": {
+            "@type": "Organization",
+            "name": "CRS4"
+          },
+          "sameAs": "https://www.crs4.it/peopledetails/219/luca-lianas"
+        },
+        {
+          "@type": "Person",
+          "name": "Paul Pfelts",
```

Felts (extra P)

---

:+1: (feel free to use suggestions)

```suggestion
          "name": "Paul Felts",
```